### DISCUSSION TITLE
输出flusher_kafka_v2中的TagFieldsRename无法修改多个字段类型

### DISCUSSION THREAD

**Discussion by q121102302** [alibaba/ilogtail#738] [Help] [answered]
<img width="278" alt="image" src="https://user-images.githubusercontent.com/101638329/226821775-a20ff323-61e7-4306-a850-5ef3975e4c93.png">
想要修改host.ip,host.name等tags中的字段。在测试中发现，host.ip总是无效的
<img width="837" alt="image" src="https://user-images.githubusercontent.com/101638329/226822160-5c6df116-74d2-4d76-9826-19fbf8e0f4aa.png">

**Comment by messixukejia** [comment]
完整的配置项发下，host.ip,host.name是怎么配置进去的？如果是自己配置的，应该用rename插件。

目前convert仅针对内置的进行转换：
![image](https://user-images.githubusercontent.com/13250519/227074366-310edd49-0777-4e85-8fbe-f6784806d2d0.png)

@shalousun 这里应该有些歧义，看看能不能优化下。

**Reply by q121102302** [reply]
我使用的版本1.3.1
全部配置：
<img width="962" alt="image" src="https://user-images.githubusercontent.com/101638329/227144531-6d7626f1-ea13-47d1-b066-01c18fdd5294.png">

**Reply by q121102302** [reply]
tags里面的数据 应该是ilogtail组装带上的 并不是我手动添加的

**Reply by shalousun** [reply]
host.ip确实没发替换，是在替换操作后面添加进去的，这个后面我们修复下，可以关注https://github.com/alibaba/ilogtail/issues/744

**Comment by messixukejia** [comment, marked as answer]
https://github.com/alibaba/ilogtail/issues/744 已合入主干
